### ISSUE TITLE
[Feature request] Right-clicking on Changed Items to copy item name to clipboard

### ISSUE THREAD

**Issue by Chiaki-aki** [xivdev/Penumbra#208]
Currently when browsing Changed Items from a specific mod or from the top tab, it gives you a list of items that will create an item link in-game when you left-click them. 
Right-click doesn't do anything there at the moment, so this could make it easier to search for the items changed by mods in fewer clicks.

**Comment by Ottermandias** [comment]
Right-clicking is a customization point for the API and does something if you have glamourer installed.
